specification/resources/vpcs/vpcs_list_peerings.yml

```
@@ -0,0 +1,41 @@
+operationId: vpcs_list_peerings
+
+summary: List the Peerings of a VPC
+
+description: |
+  To list all of a VPC's peerings, send a GET request to
+  `/v2/vpcs/$VPC_ID/peerings`.
+
+tags:
+  - VPCs
+
+parameters:
+  - $ref: 'parameters.yml#/vpc_id'
+  - $ref: '../../shared/parameters.yml#/per_page'
+  - $ref: '../../shared/parameters.yml#/page'
+
+responses:
+  '200':
+    $ref: 'responses/vpc_peerings.yml'
+
+  '401':
+    $ref: '../../shared/responses/unauthorized.yml'
+
+  '404':
+    $ref: '../../shared/responses/not_found.yml'
+
+  '429':
+    $ref: '../../shared/responses/too_many_requests.yml'
+
+  '500':
+    $ref: '../../shared/responses/server_error.yml'
+
+  default:
+    $ref: '../../shared/responses/unexpected_error.yml'
+
+x-codeSamples:
+  - $ref: 'examples/curl/vpcs_list_peerings.yml'
+
+security:
+  - bearer_auth:
+    - 'vpc:read'
```

```suggestion
    - 'vpc_peering:read'
```

Let's confirm with the team, but I think this one should be `vpc_peering:read`

---

oops, yep. updated separately before i saw this.